core/job/job.go

```diff
@@ -54,6 +59,14 @@ func (j *Job) GetName() string {
 	return j.spec.name.String()
 }
 
+func (j *Job) URN() string {
+	return fmt.Sprintf("urn:%s:%s:job:%s.%s.%s", urnContext, j.tenant.ProjectName(), j.tenant.ProjectName(), j.tenant.NamespaceName(), j.GetName())
+}
+
+func GetURN(jobName Name, tnnt tenant.Tenant) string {
+	return fmt.Sprintf("urn:%s:%s:job:%s.%s.%s", urnContext, tnnt.ProjectName(), tnnt.ProjectName(), tnnt.NamespaceName(), jobName.String())
+}
+
```

Comment:
Please remove if URN is not required here

Reply:
sure